test/components/checkbox.spec.tsx

```diff
@@ -299,6 +299,39 @@ describe('<Checkbox/>', () => {
                 expect(checkbox.nativeInput).to.have.attribute('aria-controls', '123,345');
             });
         });
+
+        it('gets focus after click (should not be in focused style state)', async () => {
+            const {driver: checkbox, waitForDom} = clientRenderer.render(
+                <CheckBox />
+            ).withDriver(CheckBoxTestDriver);
+
+            await waitForDom(() => {expect(checkbox.root).to.be.present(); });
```

Comment:
Can just write this line as: ` await waitForDom(() => expect(checkbox.root).to.be.present());`